mappings/net/minecraft/block/Block.mapping

```diff
@@ -326,15 +333,15 @@ CLASS none/aky net/minecraft/block/Block
 	METHOD j getRawIdFromBlockState (Lnone/asm;)I
 		ARG 0 state
 	METHOD k setUnbreakable ()Lnone/aky;
-	METHOD k (Lnone/asm;)Z
+	METHOD k isOpaqueFullBoundsCube (Lnone/asm;)Z
 		ARG 0 state
 	METHOD l shouldTickRandomly ()Z
-	METHOD l (Lnone/asm;)Z
+	METHOD l isFullBoundsCube (Lnone/asm;)Z
 		ARG 0 state
 	METHOD m hasBlockEntity ()Z
-	METHOD m (Lnone/asm;)I
+	METHOD m getLightOpacityLevel (Lnone/asm;)I
 		ARG 0 state
-	METHOD n (Lnone/asm;)Z
+	METHOD n passesLight (Lnone/asm;)Z
```

Comment:
`allowsLightTransmission` might be better. I feel like there's a better name still... hm